indexers/txindex/indexer_service.go

```diff
@@ -99,14 +102,33 @@ func (is *IndexerService) OnStop() {
 }
 
 // Prune removes tx and blocks indexed up to (but not including) a height.
-func (is *IndexerService) Prune(from, to uint64) error {
-	_, err := is.blockIdxr.Prune(from, to, is.Logger)
+func (is *IndexerService) Prune(to uint64, s store.Store) (uint64, error) {
+	// load indexer base height
+	indexerBaseHeight, err := s.LoadIndexerBaseHeight()
```

Comment:
seems that if we fail to loa the height we just log it and the `indexerBaseHeight` will be set to 0.
this means tha the pruning functions will be called with `from` 0 (e.g blockIndexer) (and may unncessarily iterate tons of indexes.)

Reply:
not sure what is the issue here, if not found it means it was never set and therefore it needs to start from 0 because it was never pruned...